stakingModel/contracts/UBIScheme.sol

```diff
@@ -246,11 +280,11 @@ contract UBIScheme is AbstractUBI {
         uint256 newDistribution = distributionFormula(0, account);
         activeUsersCount = activeUsersCount.sub(1);
         GoodDollar token = GoodDollar(address(avatar.nativeToken()));
-        token.transfer(msg.sender, newDistribution);
         Day memory day = claimDay[currentDay];
         day.amountOfClaimers = day.amountOfClaimers.add(1);
         day.claimAmount = day.claimAmount.add(newDistribution);
         claimDay[currentDay] = day;
```

Comment:
use storage

Reply:
this is repeating cant we extract this?

Reply:
extracted 420b81e35ef9989dca39047c7a76b2844ff4e825